subprojects/docs/src/docs/userguide/migrating_from_maven.adoc

```diff
@@ -0,0 +1,588 @@
+You simply declare the dependency in the normal way, but wrap the dependency identifier in the appropriate method, as shown in this example that "imports" the Spring Boot Dependencies BOM:
+
+.Importing a BOM in a Gradle build
+====
+include::sample[dir="userguide/mavenMigration/importBom/groovy",files="build.gradle[tags=bom]"]
+include::sample[dir="userguide/mavenMigration/importBom/kotlin",files="build.gradle.kts[tags=bom]"]
+====
+<1> Applies the Spring Boot Dependencies BOM
+<2> Adds a dependency whose version is defined by that BOM
+
+You can learn more about this feature and the difference between `platform()` and `enforcedPlatform()` in the section on <<managing_transitive_dependencies#sec:bom_import,importing version recommendations from a Maven BOM>>.
+
+NOTE: You can use this feature to apply the `<dependencyManagement>` information from any dependency's POM to the Gradle build, even those that don't have a packaging type of `pom`. However, `platform()` and `enforcedPlatform()` will ignore any dependencies declared in the `<dependencies>` block.
+
+[[migmvn:multimodule_builds]]
+== Migrating multi-module builds (project aggregation)
+
+Maven's multi-module builds map nicely to Gradle's <<multi_project_build#multi_project_build,multi-project builds>>.
+Try the corresponding link:{guidesUrl}/creating-multi-project-builds/[tutorial] to see how a basic multi-project Gradle build is set up.
+
+To migrate a multi-module Maven build, simply follow these steps:
+
+ 1. Create a settings script that matches the `<modules>` block of the root POM.
++
+For example, this `<modules>` block:
++
+[source,xml]
+----
+<modules>
+    <module>simple-weather</module>
+    <module>simple-webapp</module>
+</modules>
+----
++
+can be migrated by adding the following line to the settings script:
++
+.Declaring which projects are part of the build
+====
+include::sample[dir="userguide/mavenMigration/multiModule/groovy",files="settings.gradle"]
+include::sample[dir="userguide/mavenMigration/multiModule/kotlin",files="settings.gradle.kts"]
+====
+<1> Sets the name of the overall project
+<2> Configures two subprojects as part of this build
++
+.Output of **`gradle projects`**
+----
+> gradle projects
+include::{samplesPath}/userguide/mavenMigration/multiModule/projects.out[]
+----
+ 2. Replace cross-module dependencies with <<dependency_types#sub:project_dependencies,project dependencies>>.
+ 3. Replicate project inheritance with <<multi_project_builds#sec:cross_project_configuration,cross-project configuration>>.
++
+This basically involves creating a root project build script that injects shared configuration into the appropriate subprojects.
+
+One notable feature that's missing is a standard way to share dependency versions across projects in a multi-project build.
```

Comment:
This section needs to be updated for 5.2. The recommendation would be to use the Java Platform plugin.

Reply:
OK, but this chapter is scheduled for 5.1.